server/server.go

```diff
@@ -1,113 +1,52 @@
+	runtimeAPIVersion = "v1alpha1"
 )
 
-// Server implements the RuntimeService
-type Server struct{}
-
-// Version returns the runtime name, runtime version and runtime API version
-func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.VersionResponse, error) {
-	var err error
-
-	version, err := getGPRCVersion()
-	if err != nil {
-		return nil, err
-	}
-
-	runtimeName := "runc"
-
-	runtimeVersion, err := execRuncVersion("runc", "-v")
-	if err != nil {
-		return nil, err
-	}
-
-	runtimeApiVersion := "v1alpha1"
-
-	return &pb.VersionResponse{
-		Version:           &version,
-		RuntimeName:       &runtimeName,
-		RuntimeVersion:    &runtimeVersion,
-		RuntimeApiVersion: &runtimeApiVersion,
-	}, nil
+// Server implements the RuntimeService and ImageService
+type Server struct {
+	runtime ociRuntime
 }
 
-// CreatePodSandbox creates a pod-level sandbox.
-// The definition of PodSandbox is at https://github.com/kubernetes/kubernetes/pull/25899
-func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
-	var err error
-
-	// TODO: Parametrize as a global argument to ocid
-	ocidSandboxDir := "/var/lib/ocid/sandbox"
-	podSandboxDir := filepath.Join(ocidSandboxDir, req.GetConfig().GetName())
-
-	g := generate.New()
-
-	// TODO: Customize the config per the settings in the req
-	err = g.SaveToFile(filepath.Join(podSandboxDir, "config.json"))
-
-	return nil, err
-}
-
-// StopPodSandbox stops the sandbox. If there are any running containers in the
-// sandbox, they should be force terminated.
-func (s *Server) StopPodSandbox(context.Context, *pb.StopPodSandboxRequest) (*pb.StopPodSandboxResponse, error) {
-	return nil, nil
-}
-
-// DeletePodSandbox deletes the sandbox. If there are any running containers in the
-// sandbox, they should be force deleted.
-func (s *Server) DeletePodSandbox(context.Context, *pb.DeletePodSandboxRequest) (*pb.DeletePodSandboxResponse, error) {
-	return nil, nil
-}
-
-// PodSandboxStatus returns the Status of the PodSandbox.
-func (s *Server) PodSandboxStatus(context.Context, *pb.PodSandboxStatusRequest) (*pb.PodSandboxStatusResponse, error) {
-	return nil, nil
-}
-
-// ListPodSandbox returns a list of SandBox.
-func (s *Server) ListPodSandbox(context.Context, *pb.ListPodSandboxRequest) (*pb.ListPodSandboxResponse, error) {
-	return nil, nil
-}
-
-// CreateContainer creates a new container in specified PodSandbox
-func (s *Server) CreateContainer(context.Context, *pb.CreateContainerRequest) (*pb.CreateContainerResponse, error) {
-	return nil, nil
-}
-
-// StartContainer starts the container.
-func (s *Server) StartContainer(context.Context, *pb.StartContainerRequest) (*pb.StartContainerResponse, error) {
-	return nil, nil
+// New creates a new Server with options provided
+func New(runtimePath string) (*Server, error) {
+	// TODO(runcom): runtimePath arg is unused if we're willing to open the doors
+	// for other runtimes in the future.
```

Comment:
right, I must say: `...but it might be useful if we're willing to open the doors....` thx